commands/create_command.py

```diff
@@ -0,0 +1,29 @@
+from Icommand import ICommand
+import os, json, sys
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from SchemaKeys import SchemaKeys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self.schema = schema
+        self.data = json.load(open(schema, "r"))
+
+    def execute(self):
+        if (self.schema == None or not os.path.exists(os.path.join(parent_dir, self.schema))):
+            raise Exception("FileNotFound")
+        
+        if not SchemaKeys().DATABASE in self.data:
+            raise Exception("DatabaseNameIsMissing")
```

Comment:
I recommend creating a private function called `_validate_schema`. This validates if database and all mandatory fields exist or not.